index.js

```diff
@@ -24,10 +24,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var transpileTargets = (options && options.targets) ||
     buildTargets(options || {});
 
+  var debug = options.debug !== undefined ? options.debug : true;
```

Comment:
i think it'd actually be better to default it to `false` - ie, `var debug = typeof options.debug === 'boolean' ? ‼options.debug : false;`